src/LiquidityPool.sol

```diff
@@ -140,6 +144,13 @@ contract LiquidityPool is Auth {
         investmentManager.requestDeposit(assets, owner);
     }
 
+    function requestDepositWithPermit(uint256 assets, address owner, uint256 deadline, uint8 v, bytes32 r, bytes32 s)
```

Comment:
Maybe I'm just out of the loop, but it feels like this could use a `@dev` comment.

Reply:
Ticket created: https://github.com/centrifuge/liquidity-pools/issues/88